### PR TITLE
Eng 1252 update plugin descriptor

### PR DESCRIPTION
**Pull request by Kerruba**

### CODE REVIEW

File: vuepress/docs/next/docs/ecr/ecr-bundle-details.md [hunk 1]
```diff
@@ -74,43 +74,21 @@ following structure.
 
 ## Plugin Descriptor
 
-A plugin should be described using the [EntandoPlugin CustomResource
```
Comment (nshaw) [inline vuepress/docs/next/docs/ecr/ecr-bundle-details.md:77]: Why did you remove the reference to the CRD definition?  It seems like that would still be useful.

Comment (Kerruba) [reply vuepress/docs/next/docs/ecr/ecr-bundle-details.md:77]: Because the descriptor had changed and is not anymore like the Entando Plugin custom resource. 

Maybe we can move that link later in the documentation when I explain how the PluginDescriptor is converted to an EntandoPlugin 

What you think?

Comment (nshaw) [reply vuepress/docs/next/docs/ecr/ecr-bundle-details.md:77]: That works for me. I just liked pointing to the descriptor where it makes sense.

Comment (Kerruba) [reply vuepress/docs/next/docs/ecr/ecr-bundle-details.md:77]: I've added a small reference in another piece of the documentation, let me know if that's ok with you :+1:

**Review by nshaw — APPROVED** [review]
